cmd/sedad/cmd/rosetta_disabled.go

```diff
@@ -0,0 +1,11 @@
+//go:build !rosetta
```

Comment:
We can make this

```golang
//go:build !rosetta static
```

And the other

```golang
//go:build rosetta,!static
```

Provided there is a static build tag... 😬